cts/scheduler/xml/bundle-order-fencing.xml

```diff
@@ -128,6 +130,13 @@
         </primitive>
       </bundle>
       <bundle id="redis-bundle">
+        <meta_attributes id="redis-meta_attributes">
+          <nvpair id="redis-meta_attributes-interleave" name="interleave" value="true"/>
+          <nvpair id="redis-meta_attributes-notify" name="notify" value="true"/>
+          <nvpair id="redis-meta_attributes-ordered" name="ordered" value="true"/>
+	  <nvpair id="redis-meta_attributes-container-attribute-target"
+                  name="container-attribute-target" value="host"/>
```

Comment:
I didn't search for `container-attribute-target` since I only looked at the clone attr list. There are three more where this meta attribute is attached to the primitive instead of the bundle:
```
('bundle-probe-order-1.xml', ['galera-bundle']),
('bundle-probe-order-2.xml', ['galera-bundle']),
('bundle-probe-order-3.xml', ['galera-bundle']),
```

Reply:
Those are fine -- container-attribute-target actually is a primitive meta-attribute, but the primitive inherits from the bundle so it was easier to keep everything together